src/category_theory/limits/shapes/zero_objects.lean

```diff
@@ -148,6 +156,8 @@ localized "attribute [instance] category_theory.limits.has_zero_object.has_zero"
 lemma is_zero_zero : is_zero (0 : C) :=
 has_zero_object.zero.some_spec
 
+instance has_zero_object_op : has_zero_object Cᵒᵖ := ⟨⟨opposite.op 0, is_zero.op (is_zero_zero C)⟩⟩
```

Comment:
Do we also want the converse (not as an instance?)

Reply:
Ok, I have just added it.